src/ansys/fluent/core/workflow.py

```diff
@@ -1002,6 +1041,13 @@ def __init__(self, workflow: PyMenuGeneric, command_source: PyMenuGeneric) -> No
         self._main_thread_ident = None
         self._task_objects = {}
         self._dynamic_interface = False
+        self._help_string_command_id_map = {}
+        self._unwanted_attrs = {
+            "reset_workflow",
+            "initialize_workflow",
+            "load_workflow",
+            "create_new_workflow",
```

Comment:
```suggestion
            "reset_workflow",
            "initialize_workflow",
            "load_workflow",
            "create_new_workflow",
            "fault_tolerant",
            "part_management",
            "pm_file_management",
            "service",
            "task_object",
            "watertight",
            "workflow"]
```
I also got confused here because `rename()` is a datamodel method (I did not recognize it thus immediately) and `Rename()`/`rename()` exists at task level. So, although it is clear that `rename()` is a datamodel method here, when we are dealing with tasks we surely have a collision!

@prmukherj

Reply:
@prmukherj I just noticed that we are still exposing `get_attrib_value()` which can remove altogether from datamodel interface!